les/odr_requests.go

```diff
@@ -221,19 +214,6 @@ func (r *TrieRequest) Validate(db ethdb.Database, msg *Msg) error {
 	log.Debug("Validating trie proof", "root", r.Id.Root, "key", r.Key)
 
 	switch msg.MsgType {
-	case MsgProofsV1:
```

Comment:
Can we also remove the switch statement here and add a if check `if msg.MsgType != MsgProofsV2` then ...
Just for the code style consistency.

Reply:
done